tox.ini

```diff
@@ -1,16 +1,18 @@
 [tox]
-envlist = py27,py34,py35,py36
+envlist = py{27,34,35,36}-django{110,111,20}
 skip_missing_interpreters = True
 skipsdist = True
 
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test_requirements.txt
-commands =
-	py.test {posargs}
+    django110: Django>=1.10,<1.11
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0
+commands = py.test {posargs}
 passenv = *
 setenv =
```

Comment:
Can you remove the entire block here and update the respective environment variables in `docker-compose.yml`? Since the hostnames here are `docker-compose.yml` specific anyway it doesn't make sense for them to be in `tox.ini`